spec/views/hyrax/admin/users/index.html.erb_spec.rb

```diff
@@ -3,19 +3,38 @@
   let(:users) { [] }
 
   before do
-    (1..4).each { |i| users << build(:user, display_name: "user#{i}", email: "email#{i}@example.com", last_sign_in_at: Time.zone.now - 15.minutes, created_at: Time.zone.now - 3.days) }
+    (1..4).each { |i| users << build(:user, display_name: "user#{i}", email: "email#{i}@example.com", created_at: Time.zone.now - 3.days) }
     allow(presenter).to receive(:users).and_return(users)
     assign(:presenter, presenter)
+    allow(presenter).to receive(:show_last_access?).and_return(show_last_access)
+    render
   end
 
-  it "draws user list with all users" do
-    render
-    page = Capybara::Node::Simple.new(rendered)
-    expect(page).to have_content("Username")
-    expect(page).to have_content("Roles")
-    expect(page).to have_content("Last access")
-    (1..4).each do |i|
-      expect(page).to have_content("email#{i}@example.com")
+  let(:page) { Capybara::Node::Simple.new(rendered) }
+
+  context 'when show_last_access? is true' do
+    let(:show_last_access) { true }
+
+    it 'draws user list with all users' do
```

Comment:
This test fails because the internal test app doesn't get generated with the trackable module.  Maybe the solution here is to add more stubbing even though that does leave some code untested?

Reply:
Yep. I was using my old generated code.

Reply:
Should be fixed now.